Source/Core/PlaneGeometry.js

```diff
@@ -0,0 +1,380 @@
+                tangents[3]  = 1.0;
+                tangents[4]  = 0.0;
+                tangents[5]  = 0.0;
+                tangents[6]  = 1.0;
+                tangents[7]  = 0.0;
+                tangents[8]  = 0.0;
+                tangents[9]  = 1.0;
+                tangents[10] = 0.0;
+                tangents[11] = 0.0;
+
+                // -z face
+                tangents[12] = -1.0;
+                tangents[13] = 0.0;
+                tangents[14] = 0.0;
+                tangents[15] = -1.0;
+                tangents[16] = 0.0;
+                tangents[17] = 0.0;
+                tangents[18] = -1.0;
+                tangents[19] = 0.0;
+                tangents[20] = 0.0;
+                tangents[21] = -1.0;
+                tangents[22] = 0.0;
+                tangents[23] = 0.0;
+
+                attributes.tangent = new GeometryAttribute({
+                    componentDatatype : ComponentDatatype.FLOAT,
+                    componentsPerAttribute : 3,
+                    values : tangents
+                });
+            }
+
+            if (vertexFormat.bitangent) {
+                var bitangents = new Float32Array(2 * 4 * 3);
+
+                // +z face
+                bitangents[0] = 0.0;
+                bitangents[1] = 1.0;
+                bitangents[2] = 0.0;
+                bitangents[3] = 0.0;
+                bitangents[4] = 1.0;
+                bitangents[5] = 0.0;
+                bitangents[6] = 0.0;
+                bitangents[7] = 1.0;
+                bitangents[8] = 0.0;
+                bitangents[9] = 0.0;
+                bitangents[10] = 1.0;
+                bitangents[11] = 0.0;
+
+                // -z face
+                bitangents[12] = 0.0;
+                bitangents[13] = 1.0;
+                bitangents[14] = 0.0;
+                bitangents[15] = 0.0;
+                bitangents[16] = 1.0;
+                bitangents[17] = 0.0;
+                bitangents[18] = 0.0;
+                bitangents[19] = 1.0;
+                bitangents[20] = 0.0;
+                bitangents[21] = 0.0;
+                bitangents[22] = 1.0;
+                bitangents[23] = 0.0;
+
+                attributes.bitangent = new GeometryAttribute({
+                    componentDatatype : ComponentDatatype.FLOAT,
+                    componentsPerAttribute : 3,
+                    values : bitangents
+                });
+            }
+
+            // 4 triangles:  2 faces, 2 triangles each.
+            indices = new Uint16Array(2 * 2 * 3);
+
+            // +z face
+            indices[0] = 0;
+            indices[1] = 1;
+            indices[2] = 2;
+            indices[3] = 0;
+            indices[4] = 2;
+            indices[5] = 3;
+
+            // -z face
+            indices[6] = 4 + 2;
+            indices[7] = 4 + 1;
+            indices[8] = 4 + 0;
+            indices[9] = 4 + 3;
+            indices[10] = 4 + 2;
+            indices[11] = 4 + 0;
+        } else {
```

Comment:
What about `if (!vertexFormat.position)`?  It looks like we're creating positions regardless